File: Pemro1_Modul1/Modul1_C/PRAK110-2510817320010-PutriDeissytaHerliyana.c
#include <stdio.h>
#include <math.h>

int main() {
    float alas = 5;
    float tinggi = 12;
    float sisi_miring, keliling, luas;

    sisi_miring = sqrt(alas * alas + tinggi * tinggi);
    keliling = alas + tinggi + sisi_miring;
    luas = 0.5 * alas * tinggi;

    printf("Diketahui :\n");
    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n", tinggi);

    printf("\nJawab :\n");
    printf("Sisi A = %.0f cm\n", alas);
    printf("Sisi B = %.0f cm\n", tinggi);
    printf("Sisi C = %.0f cm\n", sisi_miring);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm\n", luas);

    return 0;
}

File: Pemro1_Modul1/Modul1_Pyton/PRAK110-2510817320010-PutriDeissytaHerliyana.py
alas = 5
tinggi = 12

sisimiring = (alas**2 + tinggi**2) ** 0.5

keliling = alas + tinggi + sisimiring
luas = (alas * tinggi) / 2

print("Diketahui :")
print("Alas =", alas, "cm")
print("Tinggi =", tinggi, "cm")

print("\nJawab :")
print(f"Sisi A = {alas:.0f} cm")
print(f"Sisi B = {tinggi:.0f} cm")
print(f"Sisi C = {sisimiring:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm")